### PR TITLE
Test

### PR DESCRIPTION
**Pull request by jburel**
Compile and run the tests
The tests should now be run via travis
To test locally, run:
``gradle test``

### CODE REVIEW

**Comment by dominikl** [comment]
Looks alright, travis passes. Just wondering if there's a way to get a more detailed output, which tests were executed etc. or is `> Task :test` and hopefully absence of error messages everything you'll get?

**Comment by jburel** [comment]
One option is to run ``gradle test -i``

File: build.gradle [hunk 1]
```diff
@@ -28,6 +28,10 @@ ext {
 
 dependencies {
     testImplementation("junit:junit:4.12")
+    testImplementation("jmock:jmock:1.+")
+    testImplementation("org.testng:testng:6.14.2")
+    testImplementation('nl.javadude.assumeng:assumeng:1.2.4')
+    testImplementation('org.openmicroscopy:omero-common-test:5.5.0-m4')
```
Comment (mtbc) [inline build.gradle]: Will this get automatically updated by our scripts? (Doesn't block this PR.)

Comment (jburel) [reply build.gradle]: I have not looked at that.

File: src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java [hunk 2]
```diff
@@ -113,14 +111,12 @@ public Object invoke(MethodInvocation arg0) throws Throwable {
...
             throws Exception {
-        IAdminPrxHelper admin = IAdminPrxHelperFactory.create();
 
         callsActiveServices(Collections.singletonList(adminServiceId));
-        map.put(adminServiceId, new _IAdminTie());
         mockAdapter.expects(once()).method("add").will(returnValue(null));
         mockAdapter.expects(once()).method("find").will(returnValue(null));
         mockAdapter.expects(once()).method("createDirectProxy").will(
-                returnValue(admin));
+                returnValue(null));
```
Comment (mtbc) [inline src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java]: That's an interesting change.

File: build.gradle (hunk 1 above)
Comment (mtbc) [reply build.gradle]: Might just mean we need a process to remember to adjust when we ought.

File: src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java (hunk 2 above)
Comment (jburel) [reply src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java]: This is one the ``blitz->server`` relationship

Comment (jburel) [reply src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java]: Sorry wrong repo

Comment (jburel) [reply src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java]: It matches the other tests

Comment (mtbc) [reply src/test/java/ome/services/blitz/test/utests/ServiceFactoryServiceCreationDestructionTest.java]: Aha, thanks.

**Comment by jburel** [comment]
Merging for m6

**Pull request merged**